conf/supervisord-dev.conf

```diff
@@ -9,6 +9,13 @@ stderr_events_enabled=true
 stopsignal = KILL
 stopasgroup = true
 
+[program:web-https]
```

Comment:
Now the supervisor conf includes a new services that happens to use the same config as `web` but with certificates and port overridden.

Reply:
:+1: If we dislike the duplication here we can deal with that later when we finally get around to applying the cookiecutter to LMS